1 table.sql

```diff
@@ -0,0 +1,38 @@
+use CostumeDB
+go
+
+
+drop table if exists dbo.Costume
+go 
+create table dbo.Costume(
+    CostumeId int not null identity primary key,
+    CustomerFirstName varchar(30) not null
+        constraint c_Costume_Customer_First_Name_cannot_be_blank check(CustomerFirstName <> ''),
+    CustomerLastName varchar(30) not null
+        constraint c_Costume_Customer_Last_Name_cannot_be_blank check(CustomerLastName <> ''),
+    Costume varchar(30) not null 
+        constraint c_Costume_Costume_can_be_AmericanGirlDoll_Artist_BumbleBee_ColonialBoy_ColonialGirl_Elephant_FireMan_PoliceMan_Princess_or_Zebra
+         check(Costume in ('American Girl Doll', 'Artist', 'Bumble Bee', 'Colonial Boy', 'Colonial Girl', 'Elephant', 'Fire Man', 'Police Man', 'Princess', 'Zebra')),
+    Size varchar(2) not null
+        constraint c_Costume_Size_can_be_XS_S_M_L_or_XL check(Size in ('XS', 'S', 'M', 'L', 'XL')),
+    AmountBought int not null 
+        constraint c_Costume_Amount_bought_must_be_1_or_more check(AmountBought >= 1),
+    SoldPricePerCostume int not null, 
```

Comment:
LB: SoldPricePerCostume is missing constraint.

Reply:
Can you be more specific about what constraint is missing?
I included constraint at end that SoldPricePerCostume must be >= BoughtPricePerCostume

Reply:
Thank you! added that constraint